packages/web3-core-types/src/types.ts

```diff
@@ -127,7 +127,7 @@ export type ProviderEventListenerMessage = (message: ProviderMessage) => void;
 export type Web3Client = string | Eip1193Provider;
 
 export interface Eip1193Provider {
-    request: (args: RequestArguments) => Promise<RpcResponse>;
+    request: (args: Eth1RequestArguments | any) => Promise<RpcResponse>;
```

Comment:
Trying to debug this right now, running into a type error if I change this to `    request: (args: Eth1RequestArguments | Eth2RequestArguments) => Promise<RpcResponse>;` 

think this has to do with `interface IWeb3Provider extends Eip1193Provider`

Error: `web3-eth2-beaconchain: src/index.ts(43,17): error TS2345: Argument of type '{ endpoint: string; params?: object | readonly unknown[] | undefined; providerOptions?: HttpOptions | undefined; returnType?: ValidTypesEnum | undefined; }' is not assignable to parameter of type 'Eth1RequestArguments'.`

Reply:
Will be addressed when eip1993 provider gets changed